corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -208,6 +215,7 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
             this.page = null;
             this.sortIndex = null;
             this.search = null;
+            this.casesPerPage = null;
```

Comment:
I don't think you need to clear `casesPerPage`. The other variables here are navigation-related and don't make sense when the user starts a new session, but I'd guess users will want `casesPerPage` to be sticky throughout web apps. In HQ we typically save items per page values to a cookie, which could be something to consider here - it's pretty easy with `$.cookie`: https://github.com/dimagi/commcare-hq/blob/655e27988e89cf0ef05ab53ed014da8c2eb92d12/corehq/apps/cloudcare/static/cloudcare/js/formplayer/users/utils.js#L17